src/gui/optionsdialog.ui

```diff
@@ -185,27 +169,44 @@
                 </spacer>
                </item>
                <item row="1" column="0">
-                <widget class="QLabel" name="label_15">
+                <widget class="QLabel" name="label_9">
                  <property name="text">
-                  <string>Theme:</string>
+                  <string>Language:</string>
+                 </property>
+                </widget>
+               </item>
+               <item row="2" column="0" colspan="2">
+                <widget class="QCheckBox" name="checkUseSystemIcon">
+                 <property name="text">
+                  <string>Use system icon theme</string>
                  </property>
                 </widget>
                </item>
                <item row="1" column="1">
-                <widget class="QComboBox" name="comboTheme"/>
+                <widget class="QComboBox" name="comboI18n">
+                 <property name="minimumSize">
+                  <size>
+                   <width>0</width>
+                   <height>0</height>
+                  </size>
+                 </property>
+                 <property name="sizeAdjustPolicy">
+                  <enum>QComboBox::AdjustToContents</enum>
+                 </property>
+                 <property name="modelColumn">
+                  <number>0</number>
+                 </property>
+                </widget>
                </item>
-               <item row="1" column="2">
-                <widget class="QLabel" name="lbl_i18n_info_3">
+               <item row="0" column="0" colspan="2">
+                <widget class="QLabel" name="label_15">
                  <property name="font">
                   <font>
                    <italic>true</italic>
                   </font>
                  </property>
                  <property name="text">
-                  <string>(Requires restart)</string>
-                 </property>
-                 <property name="alignment">
-                  <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
+                  <string>Changing Interface settings requires application restart.</string>
```

Comment:
maybe we can remove the period (punctuation mark) from the end?

Reply:
should I wait to push it 
![image](https://user-images.githubusercontent.com/34717789/72346180-69ec3380-36fb-11ea-9990-575b96ec0f76.png)

Reply:
@jagannatharjun: 

> opinion? 

Comparing the screenshots, I agree with removing the period here. 

As for the text itself, I find it somewhat long-winded, but otherwise OK.  I consider the new label a suitable compromise.  Thank you.

Reply:
done. @Chocobo1  please have a look again.